app/src/event/metricEvents.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
I would like to have non metric related markers. Just contextless events/attributes of operations as opposed to strictly metric-oriented features. It is that metric binds to code, not code to metrics :)